src/components/Navbar.css

```
@@ -0,0 +1,63 @@
+* {
+    font-family: Poppins;
+
+}
+
+.nav-main {
+    display: flex;
+    gap: 35rem;
+    padding: 2rem 2rem;
+    align-items: center;
+}
+
+.nav-content p {
+    color: #004B2F;
+    font-weight: 700;
+    size: 24px;
+}
+
+.nav-log-image {
+    width: 23px;
+    margin-right: 0px;
+}
+
+.nav-left {
+    display: flex;
+    margin-left: 3rem;
+}
+
+.nav-right ul {
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+
+}
+
+.nav-right ul li {
+    list-style: none;
+
+
+}
+
+.Home-style {
+    color: #83C727;
+    size: 16px;
+    font-weight: 600;
+}
+
+.AllStudents-style {
+    size: 16px;
+    color: #83C727;
+    font-weight: 600;
+    background-color: #FFFFFF;
+    border: 2px solid #83C727;
+    border-radius: 8px;
+}
+
+.AddStudent-style {
+    color: #686868;
+    font-weight: 500;
+    size: 16px;
+
+
```

Use kebab-case for CSS class names instead of camelCase for better consistency with CSS conventions.

```suggestion
.home-style {
    color: #83c727;
    font-size: 16px;
    font-weight: 600;
}

.all-students-style {
    font-size: 16px;
    color: #83c727;
    font-weight: 600;
    background-color: #ffffff;
    border: 2px solid #83c727;
    border-radius: 8px;
}

.add-student-style {
    color: #686868;
    font-weight: 500;
    font-size: 16px;
}
```